Libraries/TranscriptTestRunner/TestRunner.cs

```diff
@@ -98,10 +98,17 @@ public async Task<Activity> GetNextReplyAsync(CancellationToken cancellationToke
             }
         }
 
-        public async Task AssertReplyAsync(Action<Activity> validateAction, CancellationToken cancellationToken = default)
+        protected virtual void AssertReply(TestScript expectedActivity, Activity actualActivity)
```

Comment:
Let's make this guy Async and add cancellation token in case the assertion involves IO (e.g.: calling some external service or querying a database).
Also let's call this method AssertActivityAsync instead of AssertReplyAsync

Reply:
Feedback applied. Thanks!